policies/stable-release-updates.md

```diff
@@ -46,6 +46,11 @@ existing public specifications that we claim to conform to. Where the
 documentation explicitly contradicts the specifications the documented behavior
 is what matters.
 
+A **bug fix** is also a fix of a current or near future FIPS compliance
+requirement in the FIPS module code if it does not break existing use-cases.
```

Comment:
`if it does not break existing use-cases` looks like too strict here.

Reply:
Why? This is what we discussed last week on OTC AFAIK. If we need to break it we can still explicitly give exception.

But I assume in most if not all cases we can simply make it configurable.